style.css

```diff
@@ -0,0 +1,102 @@
+.wrapper {
+	width: 80%;
+	margin: auto;
+	border: dashed 1px red;
+}
+
+.header {
+	display: flex;
+	justify-content: space-between;
+}
+
+.header__logo {
+	padding: 40px 0;
+	font-family: 'ptserif-bold', arial, helvetica, Sans-serif;
+	font-weight: bold;
+	font-size: 24px;
+	line-height: 31px;
+	position: relative;
+	padding-left: 44px;
+}
+
+.logo__circle {
+	position: absolute;
+	left: 0;
+	top: 50%;
+	transform: translateY(-50%);
+	width: 28px;
+	height: 28px;
+	border: dashed 1px red;	
+}
+
+.logo__circle_color_dark-blue {
+	background: #0056EC;
+	border-radius: 50%;
+	opacity: .7;
+}
+
+.logo__circle_color_blue {
+	left: 8px;
+	top: 56%;
+	background: #C8DBFB;
+	opacity: .7;
+	border-radius: 50%
+}
+
+.menu {
+	display: flex;
+	align-items: center;
+}
+
+.menu__item {
+	padding: 0 13px;
+	text-decoration: none;
+	font-family: Roboto;
+	font-style: normal;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)